src/R6RegionChanger/clsScript.vb

```diff
@@ -132,75 +132,87 @@ Public Class clsScript
                     R6FullProfilePathFI.IsReadOnly = True
 
-                ElseIf frmMain.dropRegion.Text = "neu (Europe North)" Then
-                    writeIniValue("ONLINE", "DataCenterHint", "neu", R6FullProfilePath)
+                ElseIf frmMain.dropRegion.Text = "northeurope (Europe North)" Then
+                    writeIniValue("ONLINE", "DataCenterHint", "playfab/northeurope", R6FullProfilePath)
                     frmMain.btnApply.Enabled = False
-                    MessageBox.Show("Settings saved successfully!" & vbNewLine & vbNewLine & "Profile: " & frmMain.dropProfile.Text & vbNewLine & "Region: neu" & vbNewLine & "Read-only: " & frmMain.cbReadOnly.Checked, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information)
+                    MessageBox.Show("Settings saved successfully!" & vbNewLine & vbNewLine & "Profile: " & frmMain.dropProfile.Text & vbNewLine & "Region: northeurope (Europe North)" & vbNewLine & "Read-only: " & frmMain.cbReadOnly.Checked, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information)
                     R6FullProfilePathFI.IsReadOnly = True
 
-                ElseIf frmMain.dropRegion.Text = "weu (Europe West)" Then
-                    writeIniValue("ONLINE", "DataCenterHint", "weu", R6FullProfilePath)
+                ElseIf frmMain.dropRegion.Text = "westeurope (Europe West)" Then
+                    writeIniValue("ONLINE", "DataCenterHint", "playfab/westeurope", R6FullProfilePath)
                     frmMain.btnApply.Enabled = False
-                    MessageBox.Show("Settings saved successfully!" & vbNewLine & vbNewLine & "Profile: " & frmMain.dropProfile.Text & vbNewLine & "Region: weu" & vbNewLine & "Read-only: " & frmMain.cbReadOnly.Checked, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information)
+                    MessageBox.Show("Settings saved successfully!" & vbNewLine & vbNewLine & "Profile: " & frmMain.dropProfile.Text & vbNewLine & "Region: westeurope (Europe West)" & vbNewLine & "Read-only: " & frmMain.cbReadOnly.Checked, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information)
+                    R6FullProfilePathFI.IsReadOnly = True
+                    
+                ElseIf frmMain.dropRegion.Text = "southafricanorth (africa south north)" Then
+                    writeIniValue("ONLINE", "DataCenterHint", "playfab/southafricanorth", R6FullProfilePath)
+                    frmMain.btnApply.Enabled = False
+                    MessageBox.Show("Settings saved successfully!" & vbNewLine & vbNewLine & "Profile: " & frmMain.dropProfile.Text & vbNewLine & "Region: southafricanorth (africa south north)" & vbNewLine & "Read-only: " & frmMain.cbReadOnly.Checked, "Information", MessageBoxButtons.OK, MessageBoxIcon.Information)
```

Comment:
@danielwerg `dropRegion.Text` is case-sensitive, this does not work (string in droplist and in this elif condition are not the same).

The string in the MsgBox for the region should reflect the string in the droplist.

#2